README.md

```diff
@@ -1,16 +1,20 @@
 # Node Version Manager [![Build Status](https://travis-ci.org/nvm-sh/nvm.svg?branch=master)][3] [![nvm version](https://img.shields.io/badge/version-v0.35.1-yellow.svg)][4] [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/684/badge)](https://bestpractices.coreinfrastructure.org/projects/684)
 
+<!-- To update this table of contents, ensure you have run npm/yarn then run doctoc -->
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 ## Table of Contents
 
-- [Installation and Update](#installation-and-update)
```

Comment:
this means the link for `#installation-and-update` will be broken, let's avoid that by adding it as an explicit `<a id="installation-and-update"></a>` - or, by reverting the section name change.

Reply:
Is this to say that the url `.../nvm-sh/nvm/README.md#installation-and-update` is used somewhere other than the readme document? 

If not, I'd suggest the references within the document are just updated accordingly.

If there is then I'm happy to revert or add an anchor (preference for adding a backwards compatible anchor though).

Reply:
All URLs should always be expected to be used outside the document, and thus no URL that has ever existed should ever be removed.

Reply:
It took me a mo for my brain to catch up but got what you mean, and have resolved this